packages/beacon-node/src/chain/blocks/types.ts

```diff
@@ -21,63 +21,94 @@ export enum BlockSource {
   byRoot = "req_resp_by_root",
 }
 
+export enum GossipedInputType {
+  block = "block",
+  blob = "blob",
+  dataColumn = "dataColumn",
```

Comment:
```suggestion
  dataColumn = "data_column",
```

I think it looks nice that way if we render on grafana

Reply:
updated

Reply:
updated